source/world_builder/features/plume_models/temperature/gaussian.cc

```diff
@@ -0,0 +1,166 @@
+          // Add `max distance fault` center to the required parameters.
+          prm.declare_entry("", Types::Object({"centerline temperatures"}),
+                            "Gaussian temperature model. The temperature is interpolated between the plume center "
+                            "and margin (as defined by the plume feature) using a Gaussian function: "
+                            "T(r) = T_center(z) exp(-r^2/(2 sigma^2). "
+                            "The temperature at the plume centerline T_center can be changed with depth by defining "
+                            "an array of depths and centerline temperatures, and temperature is interpolated linearly "
+                            "with depth. Similarly, the sigma of the Gaussian function (relative to the width of "
+                            "the plume as given by the plume feature) can be changed with depth. "
+                            "Temperature is always interpolated in a horizonzal/radial plane, except for the plume "
+                            "head: If the first depth of the plume centerline and the minimum depth of the plume "
+                            "feature are different, an ellipsoidal plume head is created in this depth range. "
+                            "Within this plume head, temperature is interpolated radially, i.e., depending on the "
+                            "distance from the center of the ellipsoid.");
+
+          prm.declare_entry("depths", Types::Array(Types::Double(0)),
+                            "The temperature at the center of this feature in degree Kelvin."
+                            "If the value is below zero, the an adiabatic temperature is used.");
+          prm.declare_entry("centerline temperatures", Types::Array(Types::Double(0)),
+                            "The temperature at the center of this feature in degree Kelvin."
+                            "If the value is below zero, the an adiabatic temperature is used.");
+          prm.declare_entry("gaussian sigmas", Types::Array(Types::Double(0.3)),
+                            "The sigma (standard deviation) of the Gaussian function used to compute the "
+                            "temperature distribution within the plume. This sigma is non-dimensinal, i.e. "
+                            "it is defined relative to the distance between the plume center and margin as "
+                            "defined by the plume feature. Choosing a sigma of 1 therefore means that the "
+                            "temperature at the plume margin is set to a fraction of 1/sqrt(e) (approx. 0.61) "
+                            "of the centerline temperature. To achieve a smoother transition between the "
+                            "plume temperature and the outside temperature a smaller values has to be chosen "
+                            "for the gaussian sigmas.");
+
+          // TODO: assert that the three have the same length
```

Comment:
This todo belongs in parse_entries, and it looks like it is already done. I think you can just remove this.